event_sourcery/event_store/tenant_id.py

```diff
@@ -0,0 +1,5 @@
+from typing import Final, TypeAlias
+
+TenantId: TypeAlias = str
+
+DEFAULT_TENANT: Final[TenantId] = "*default*"
```

Comment:
This is ok with you @lzukowski ? I remember you had some complaints about asterisks 🤔

Reply:
It's ok in my code, but ut can impact the usage of ESDB, cause it's impacting regex.

Reply:
changed